app/controllers/gyms_controller.rb

```diff
@@ -19,13 +23,35 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @gym.update(gym_params)
+      flash[:notice] = 'Academia atualizada com sucesso!'
```

Comment:
Só pra fins de curiosidade com redirect você também pode fazer assim. Mas só funciona com redirect com render não. Também só pode ser feito com os flashes padrão notice e alert

```suggestion
      redirect_to @gym, notice: 'Academia atualizada com sucesso!'
```

Reply:
Algumas linhas foram poupadas, muito bom saber disso, obrigada pela dica!